ethereum/contracts/pyth/PythState.sol

```diff
@@ -38,11 +38,15 @@ contract PythStorage {
 
         // Mapping of cached price information
         // priceId => PriceInfo
-        mapping(bytes32 => PythInternalStructs.PriceInfo) latestPriceInfo;
+        mapping(bytes32 => PythInternalStructs.PriceInfo) _anotherDeprecatedLatestPriceInfo;
```

Comment:
Shall we rename the PriceInfo structs V1,V2,V3.. etc, which would allow us to call these deprecated mapping fields something more meaningful like `deprecatedLatestPriceInfoV1`, `deprecatedLatestPriceInfoV2` etc?

Reply:
Done.